data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -754,5 +754,176 @@
     "name": "dao +2",
     "proportional": { "price": 6.0, "bashing": 1.2, "cutting": 1.2, "weight": 0.8 },
     "relative": { "to_hit": 2 }
+  },
+  {
+    "id": "rune_biomancer_weapon",
+    "type": "GENERIC",
+    "name": "Biomancer spear",
+    "plural": "Biomancer spears",
+    "\\": "Effectively a durable stone spear +1 with IMPALE tech. based off Wicked Bonespear. Lorewise, it would make sense for novice Biomancers to be developing their skills in spears so they can adequately use the Bonespear(No whips/REACH3 for you!) once they're proficient enough to summon one. Should be weaker than other school weapons DPSwise due to reach.",
```

Comment:
```suggestion
    "//": "Effectively a durable stone spear +1 with IMPALE tech. based off Wicked Bonespear. Lorewise, it would make sense for novice Biomancers to be developing their skills in spears so they can adequately use the Bonespear(No whips/REACH3 for you!) once they're proficient enough to summon one. Should be weaker than other school weapons DPSwise due to reach.",
```